src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -851,6 +851,11 @@ export class SortFilterBar
       height: 14px;
     }
 
+    .sort-direction-icon > svg {
+      height: -webkit-fit-content;
+      height: fit-content;
```

Comment:
I found `flex: 1` seemed to do the trick as well on iPad + safari ; that might be a cleaner approach? But the provided does work, so whatever works!

Reply:
Oh, you're right! That does seem more straightforward.